code/modules/ghostroles/spawner/simplemob/maintdrone.dm

```diff
@@ -2,6 +2,7 @@
 	short_name = "maintdrone"
 	name = "Maintenence Drone"
 	desc = "Maintain and Improve the Systems on the Aurora"
+	show_on_job_select = FALSE
```

Comment:
Do this for ERT too.

Reply:
nah, i don't think removing them is a good idea, as discussed on discord